tensorflow_probability/examples/jupyter_notebooks/autoregressive_flow_on_jax.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>I think Haiku made a release the other day, could we install it from Pypi?</p><p> </p><p>Also, you don't use the psd_kernels anywhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/probability/pull/1176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>